modules/microshift-nw-ipv6-single-stack-config.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * microshift_configuring/microshift-using-config-tools.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-configuring-ipv6-single-stack-config_{context}"]
+= Configuring IPv6 single-stack
+
+You can use the IPv6 network protocol by updating the {microshift-short} service configuration file.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have root access to the cluster.
+* Your cluster uses the OVN-Kubernetes network plugin.
+//if you have multus, would you also use the same procedure?
+
+//* The cluster node has IPv6 addresses.
+//also does not need to be pre-config, because we are doing that here, correct?
+//* You have configured an IPv6-enabled router based on your infrastructure.
+//microshift users don't need to pre-config, correct?
+
+.Procedure
+
+. If you have not done so, make a copy of the provided `config.yaml.default` file in the `/etc/microshift/` directory, renaming it `config.yaml`. Keep the new {microshift-short} `config.yaml` you create in the `/etc/microshift/` directory. The new `config.yaml` is read whenever the {microshift-short} service starts. After you create it, the `config.yaml` file takes precedence over built-in settings.
+//Q: would we add two sections for dual stack?
```

Comment:
Dual stack has a different way of configuring so I would add it in a different section, yes.